test/tensor/Jamfile

```diff
@@ -26,17 +26,23 @@ explicit unit_test_framework ;
 test-suite boost-ublas-tensor-test
     :
     [ run test_tensor.cpp 
-          test_strides.cpp 
-          test_operators_comparison.cpp 
-          test_operators_arithmetic.cpp
-          test_multiplication.cpp
-          test_multi_index_utility.cpp
-          test_multi_index.cpp
-          test_functions.cpp
-          test_extents.cpp
-          test_expression_evaluation.cpp
-          test_einstein_notation.cpp
-          test_algorithms.cpp
-          test_tensor_matrix_vector.cpp
+          #test_strides.cpp 
+          #test_operators_comparison.cpp 
+          #test_operators_arithmetic.cpp
+          #test_multiplication.cpp
+          #test_multi_index_utility.cpp
+          #test_multi_index.cpp
+          #test_functions.cpp
+          #test_extents.cpp
+          #test_expression_evaluation.cpp
+          #test_einstein_notation.cpp
+          #test_algorithms.cpp
+          #test_tensor_matrix_vector.cpp
+          #test_extents_helper.cpp
+          #test_basic_shape.cpp
+          test_static_extents.cpp
+          #test_static_strides.cpp
           unit_test_framework ]
+    :
+    : <define>BOOST_UBLAS_TENSOR_TESTING
```

Comment:
What is your plan here ? To re-enable these tests incrementally as you make progress on this PR ? Migrate them elsewhere ? Or disable them entirely ?

Reply:
I was trying to reduce the use of exceptions which is costly using asserts but for testing, I could not use asserts as they abort the program. Instead, I used the macro to define two different behavior, one for testing ( exception ) and another for assert. To switch between them I was using this macro. I'm still testing this.

Reply:
Tests were disabled because of MSVC. Which is giving some errors which I'm trying fix.

Reply:
OK, no worries. But please mark the PR as work-in-progress, so reviewers know what to expect.

Reply:
> I was trying to reduce the use of exceptions which is costly using asserts but for testing, I could not use asserts as they abort the program.

How are exceptions more costly?

Reply:
Resolving conversation as file is outdated.